components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -61,6 +61,20 @@ public class SubscriptionServices : ISubscriptionManager, IDisposable {
             subscriptions.ForEach(s => Try.Op(() => s.Dispose()));
         }
 
+        /// <summary>
+        /// Function to retrieve display name for a field in the select clause
+        /// </summary>
+        /// <param name="monitoredItem"></param>
+        /// <param name="index"></param>
+        /// <returns>The display name, if defined</returns>
+        public static string GetFieldDisplayName(MonitoredItem monitoredItem, int index) {
```

Comment:
Suggestion, this could be something like:

```cs
return (monitoredItem.Handle as MonitoredItemWrapper)?.GetFieldDisplayName(index);
```

Reply:
Good suggestion, fixed